modules/examiner/test/ExaminerTest.php

```diff
@@ -41,10 +41,11 @@ class ExaminerTest extends LorisIntegrationTest
             'Examiner'         => '#bc2 > a:nth-child(2) > div',
             'Selection Filter' => '#lorisworkspace > div.row > '.
                                   'div.col-sm-12.col-md-7 > div > div.panel-heading',
-            'Add Examiner'     => '#lorisworkspace > div.row > div.col'.
-                                  '-sm-12.col-md-5 > div > div.panel-heading',
+            'Add Examiner'     => '#lorisworkspace > div > div:nth-child(1) > '.
+                                  'div > div:nth-child(1)',
             'Add'              => '#examiner > div:nth-child(3) > div > button',
            );
+
```

Comment:
remove 86-90 line, because you have changed the UI, "Add Examiner" is not the button, it the original Add Examiner form.

Reply:
before
![screen shot 2019-01-17 at 10 55 53 am](https://user-images.githubusercontent.com/9876007/51330949-9e74d300-1a46-11e9-83d9-d14d678449f6.png)
after
![screen shot 2019-01-17 at 10 51 13 am](https://user-images.githubusercontent.com/9876007/51330952-a3d21d80-1a46-11e9-821b-0d925f7e16d1.png)